docs/docs/zh/guide/task/java.md

```diff
@@ -22,25 +22,39 @@
 | 模块路径     | 开启使用JAVA9+的模块化特性，把所有资源放入--module-path中，要求您的worker中的JDK版本支持模块化 |
 | 主程序参数    | 作为普通Java程序main方法入口参数                                          |
 | 虚拟机参数    | 配置启动虚拟机参数                                                     |
-| 脚本       | 若使用JAVA运行类型则需要编写JAVA代码。代码中必须存在public类，不用写package语句            |
+| 主程序包     | 选择要运行程序的主程序包                                                  |
 | 资源       | 可以是外部JAR包也可以是其他资源文件，它们都会被加入到类路径或模块路径中，您可以在自己的JAVA脚本中轻松获取      |
 
 ## 任务样例
 
-java任务类型有两种运行模式，这里以JAVA模式为例进行演示。
+java任务类型有两种运行模式，这里进行分别进行演示。
 
 主要配置参数如下：
 
 - 运行类型
 - 模块路径
 - 主程序参数
 - 虚拟机参数
-- 脚本文件
+- 主程序包
+- 资源文件
 
-![java_task](../../../../img/tasks/demo/java_task02.png)
+如图所示
+
+- FAT_JAR类型
+
+![java_task](../../../../img/tasks/demo/java_fat.png)
+
+`FAT_JAR`即`uber-jar`，依赖和代码都在同一个`jar`中，只需选择这一个`jar`即可
+
+- NORMAL_JAR类型
+
+![java_task](../../../../img/tasks/demo/java_normal.png)
+
+`normal1.jar`是程序运行的入口，`normal2.jar`是所需的依赖，需要用主程序包指定程序的入口，并且在资源文件中选择所有依赖和程序入口的`jar`文件，才能正确运行
 
 ## 注意事项
 
-使用JAVA运行类型时代码中必须存在public类，可以不写package语句
+1. 使用`FAT_JAR`运行类型必须选择主程序包，使用和对应的资源文件，否则任务会直接失败
+2. 使用`NORMAL_JAR`运行类型必须选择主程序包和对应的资源文件，否则任务会直接失败。主程序包决定程序从哪里开始运行，选择资源文件之后，运行程序时，资源文件会以`-cp`的方式进行调用
```

Comment:
Can we simplify this? The normal jar should provide main class, fat jar don't need to provide main class.

Reply:
It has been modified.